bench_test.go

```diff
@@ -15,16 +15,16 @@ func TestSimpleReadWrite(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	config := &Config{
```

Comment:
Is this to fix a race inside `pion/dtls`? 

I would rather we copy values, I think our raciness would be a surprise to users.

Reply:
Yes! This test sometimes fails when the race condition is detected by go. Though after some thought, I am going to pull this out into a separate PR so it doesn't block this PR